internal/changelog/renderer.go

```diff
@@ -132,13 +134,23 @@ func (r Renderer) Render() error {
 	return afero.WriteFile(r.fs, outFile, data.Bytes(), changelogFilePerm)
 }
 
-//go:embed asciidoc-template.asciidoc
-var asciidocTemplate embed.FS
-
 func (r Renderer) Template() ([]byte, error) {
-	data, err := asciidocTemplate.ReadFile("asciidoc-template.asciidoc")
+	var data []byte
+	var err error
+
+	embeddedFileName, ok := assets.GetEmbeddedTemplates()[r.templ]
+	if ok {
```

Comment:
Nit: I suggest to use a more idiomatic syntax here, as `embeddedFileName` is not used outside of the `if ok {` block:

```suggestion
	if embeddedFileName, ok := assets.GetEmbeddedTemplates()[r.templ]; ok {
```